config/envs/development.py

```diff
@@ -9,6 +9,6 @@
 
 @configclass
 class DevelopmentConfig(DefaultConfig):
-
+    ALLOW_HTTP_API_CALLS = True
```

Comment:
Whereabouts is this used? Is this the flag to use local data in get_data? We should have a consistent name for this switch across repos maybe?

Reply:
its not, good spot - its a hangover from a different solution to handling local data calls